client/my-sites/customer-home/cards/features/support/search-card.jsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { identity, includes } from 'lodash';
+import debugFactory from 'debug';
+
+/**
+ * Internal Dependencies
+ */
+import { recordTracksEvent } from 'state/analytics/actions';
+import SearchCard from 'components/search-card';
+import {
+	getInlineHelpCurrentlySelectedLink,
+	getSelectedResultIndex,
+	isRequestingInlineHelpSearchResultsForQuery,
+	getInlineHelpCurrentlySelectedResult,
+} from 'state/inline-help/selectors';
+import {
+	requestInlineHelpSearchResults,
+	selectNextResult,
+	selectPreviousResult,
+} from 'state/inline-help/actions';
+
+/**
+ * Module variables
+ */
+const debug = debugFactory( 'calypso:customer-home-help' );
+
+class SupportSearchCard extends Component {
+	static propTypes = {
+		openResult: PropTypes.func.isRequired,
+		translate: PropTypes.func,
+		query: PropTypes.string,
+	};
+
+	static defaultProps = {
+		translate: identity,
+		query: '',
+	};
+
+	onKeyDown = ( event ) => {
+		// ignore keyboard access when manipulating a text selection in input etc.
+		if ( event.getModifierState( 'Shift' ) ) {
+			return;
+		}
+		// take over control if and only if it's one of our keys
+		if ( includes( [ 'ArrowUp', 'ArrowDown', 'Enter' ], event.key ) ) {
+			event.preventDefault();
+		} else {
+			return;
+		}
+
+		switch ( event.key ) {
+			case 'ArrowUp':
+				this.props.selectPreviousResult();
+				break;
+			case 'ArrowDown':
+				this.props.selectNextResult();
+				break;
+			case 'Enter': {
+				const hasSelection = this.props.selectedResultIndex >= 0;
+				hasSelection && this.props.openResult( event, this.props.selectedResult );
+				break;
+			}
+		}
+	};
+
+	onSearch = ( searchQuery ) => {
+		debug( 'search query received: ', searchQuery );
+		this.props.recordTracksEvent( 'calypso_customer_home_support_search', {
```

Comment:
Should we keep the same event name that the name defined by the InlineHelperPopover and add a new event property to differentiate them, instead of creating a new event name? Asking because [we have an issue](https://github.com/Automattic/wp-calypso/issues/42105) that suggests adding this property.

Reply:
Right good. Let's add the additional property as you suggest. Do we do that in this PR do you think?

Reply:
100% a new property, not a new name. In any PR is good, I think :)

Reply:
@retrofox I think we should consider that all Home stats are expected to have a certain prefix. See `PCYsg-q5t-p2`.

Reply:
> Do we do that in this PR do you think?

[Just created](https://github.com/Automattic/wp-calypso/pull/42648). branched off from this one.

>  I think we should consider that all Home stats are expected to have a certain prefix

Good 👍  For now, I think we aren't going to do any new event but adding a new property to each event in order to be able to segment them according to its `location`. Does it make sense?

Reply:
Data prefers more generic events, so that sounds about right @retrofox (folks are of course welcome to consult data peeps on what makes more sense). Maybe one event for search and one for click max with a property of what they searched/clicked on